types/physijs/index.d.ts

```diff
@@ -1,250 +0,0 @@
-// Type definitions for Physijs
```

Comment:
How is this deletion related to the main change?

Reply:
This is a question for @Methuselah96

Reply:
Hey, sorry for the late reply. I just asked a question about this on Discord.

> Is there a way to have subfolders for old versions that that are still in the 0.x.y releases? The use case is three.js. physijs relies on @types/three@^0.93.31, but the current version is 0.125.x. The only way that I can think to deal with it is to remove the types for that package, but that doesn't seem ideal (however the package hasn't been updated in about 6 years and gets about 6 weekly downloads so I'm not sure it's actually getting used).

Reply:
Talked to @sandersn about this on Discord. Since the package is practically dead, he said that it was fine to go ahead and delete the types and add it to `notNeededPackages.json` (for the purposes of getting the build to pass).

Reply:
Oh, wait it's actually accurate. `physijs` added a definition file in April: https://github.com/loperd/physijs.

Reply:
Shouldn't `physisjs` be updated to depends on updated `threejs` version: So when it's ships to NPM (with dts changes) should fix that problem with client consumption of types:
https://github.com/loperd/physijs/blob/master/package.json#L7

Reply:
I'm not sure I follow. When the maintainers of `physijs` upgrade to a higher version of `three` that no longer has the types, they'll have to add a dependency on `@types/three`, but since they're still depending on a version of `three` that contains types it should be fine for now.

What exactly are you proposing?